modules/smartadserverBidAdapter.js

```diff
@@ -148,7 +170,9 @@ export const spec = {
           currency: response.currency,
           netRevenue: response.isNetCpm,
           ttl: response.ttl,
-          dspPixels: response.dspPixels
+          dspPixels: response.dspPixels,
+          // WE DON'T FULLY SUPPORT THIS ATM - will always return empty array until implemented on our side.
```

Comment:
I wouldn't add this comment, because when we will support this (server side) we won't think about removing this line :p